inductiva/molecules/simulators/gromacs.py

```diff
@@ -16,50 +16,40 @@ def api_method_name(self) -> str:
 
     def run(
         self,
-        input_dir: types.Path,
-        sim_config_filename: str,
-        protein_filename: str,
-        topology_filename: str,
-        output_dir: Optional[types.Path] = None,
+        input_directory: types.Path,
```

Comment:
Change the docstring on the class, since now you are able to perform any GROMACS comand.